include/functions/math.hpp

```diff
@@ -30,4 +30,42 @@ BigInt big_pow10(size_t exponent) {
     return BigInt("1" + std::string(exponent, '0'));
 }
 
+/*
+    pow
+    ---
+    Return a BigInt equal to my_big_int^x
+    NOTE: x could be BigInt, long long, or string
+*/
+BigInt pow(BigInt my_big_int, const BigInt x) {
```

Comment:
* I made a mistake while mentioning the parameters for `pow` (sorry!). I've updated them now. It should be
  ```C++
  (const T& base, const long long& exponent)
  ```
  where the type `T` is `BigInt`, `long long` and `std::string`.
* The exponent should only be an integer (`long long`), as I can't seem to think of situations where a `BigInt` would need to be raised to an exponent greater than 2<sup>63</sup> - 1 (it would require > 86 [EiB](https://en.wikipedia.org/wiki/Exbibyte) of RAM!).

Reply:
Sure, on it